media-source-respec.html

```diff
@@ -366,42 +369,47 @@
 
       <p>Each <a>MediaSource</a> object has a <dfn id="live-seekable-range">live seekable range</dfn> variable that stores a <a def-id="normalized-timeranges-object"></a>.  This variable is initialized to an empty <a def-id="timeranges"></a> object when the <a>MediaSource</a> object is created, is maintained by <a def-id="setLiveSeekableRange"></a> and <a def-id="clearLiveSeekableRange"></a>, and is used in <a href="#htmlmediaelement-extensions">HTMLMediaElement Extensions</a> to modify <a def-id="hme-seekable"></a> behavior.</p>
 
-      <dl title="enum ReadyState" class="idl">
-        <dt>closed</dt>
-        <dd>
+      <div><pre class="idl">enum ReadyState {
```

Comment:
The enum ReadyState is linked to the readyState attribute description, and that attribute (lower down) has no link.